main.js

```diff
@@ -83,6 +83,15 @@ Trello.prototype.addListToBoard = function (boardId, name, callback) {
     return makeRequest(rest.post, this.uri + '/1/boards/' + boardId + '/lists', {query: query}, callback);
 };
 
+Trello.prototype.addMemberToBoard = function (boardId, memberId, type, callback) {
+    var query = this.createQuery();
+    var data = {};
+
+    data.type = type; // Valid Values: 'normal','admin','observer'
```

Comment:
If these are the only allowed values, then we could validate that the input against them and error out early before actually making the request.